enterprise_gateway/itests/src/R_client_testcase_pytest.py

```diff
@@ -0,0 +1,47 @@
+from elyra_client import ElyraClient
```

Comment:
Is the suffix of `_testcase_pytest` a convention used by the framework?  If not, I suggest shortening it.  Seems like `_pytest` is a little more redundant than `_testcase`, so I suppose the latter could be retained.

Reply:
Negative, just a name I came up with to distinguish between older unittest files and newer pytest files. Will change with shorter and more succinct names.